.openshift-ci/build/install-gcloud-sdk.sh

```diff
@@ -1,20 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
it's still used in the drivers docker files I was just consolidating two different implementations of the install script

Reply:
I must be blind, where is the consolidated script?

Reply:
it's `.openshift-ci/scripts/gcloud-sdk-install.sh` which existed before this PR - IIRC I kept that version over the one in `.openshift-ci/driver/scripts` because it was slightly more robust